### PR TITLE
feat(octavia): prepare health monitor dashboard and delete page

### PR DESCRIPTION
**Pull request by qpavy**
ref: MANAGER-12953

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/octavia-health-monitor`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12953
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Prepare octavia health monitor dashboard and deletion page

### CODE REVIEW

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js [hunk 1]
```diff
@@ -6,13 +6,14 @@ import service from './service';
 
 import create from './create';
 import edit from './edit';
+import deletion from './delete';
```
Comment (anooparveti) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js:9]:
```suggestion
import delete from './delete';
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js [hunk 2]
```diff
@@ -6,13 +6,14 @@ import service from './service';
...
 
 import healthMonitorComponents from './components';
 
 const moduleName = 'ovhManagerOctaviaLoadBalancerPoolsDetailHealthMonitor';
 
 angular
-  .module(moduleName, [healthMonitorComponents, create, edit])
+  .module(moduleName, [healthMonitorComponents, create, edit, deletion])
```
Comment (anooparveti) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js:16]:
```suggestion
  .module(moduleName, [healthMonitorComponents, create, edit, delete])
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js (hunk 1 above)
Comment (qpavy) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js:9]: delete is a javascript reserved word so better use deletion

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js (hunk 2 above)
Comment (qpavy) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js:16]: Same as https://github.com/ovh/manager/pull/10764#discussion_r1464867109

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**